config/samples.tsv

```diff
@@ -1,5 +1,5 @@
 replicateName	sampleName	path_to_R1_fastq	path_to_R2_fastq
-D4_Meso_iCre_Dox_1	D4_Meso_iCre_Dox	PIPELINE_HOME/.test/rawdata/D4_Meso_iCre_Dox_1.subset.R1.fastq.paired.fq.gz	PIPELINE_HOME/.test/rawdata/D4_Meso_iCre_Dox_1.subset.R2.fastq.paired.fq.gz
-D4_Meso_iCre_Dox_2	D4_Meso_iCre_Dox	PIPELINE_HOME/.test/rawdata/D4_Meso_iCre_Dox_2.subset.R1.fastq.paired.fq.gz	PIPELINE_HOME/.test/rawdata/D4_Meso_iCre_Dox_2.subset.R2.fastq.paired.fq.gz
-iCre_D0_1	iCre_D0	PIPELINE_HOME/.test/rawdata/iCre_D0_1.subset.R1.fastq.paired.fq.gz	PIPELINE_HOME/.test/rawdata/iCre_D0_1.subset.R2.fastq.paired.fq.gz
-iCre_D0_2	iCre_D0	PIPELINE_HOME/.test/rawdata/iCre_D0_2.subset.R1.fastq.paired.fq.gz	PIPELINE_HOME/.test/rawdata/iCre_D0_2.subset.R2.fastq.paired.fq.gz
+D4_Meso_iCre_Dox_1	D4_Meso_iCre_Dox	${PIPELINE_HOME}/.test/rawdata/D4_Meso_iCre_Dox_1.subset.R1.fastq.paired.fq.gz	${PIPELINE_HOME}/.test/rawdata/D4_Meso_iCre_Dox_1.subset.R2.fastq.paired.fq.gz
+D4_Meso_iCre_Dox_2	D4_Meso_iCre_Dox	${PIPELINE_HOME}/.test/rawdata/D4_Meso_iCre_Dox_2.subset.R1.fastq.paired.fq.gz	${PIPELINE_HOME}/.test/rawdata/D4_Meso_iCre_Dox_2.subset.R2.fastq.paired.fq.gz
+iCre_D0_1	iCre_D0	${PIPELINE_HOME}/.test/rawdata/iCre_D0_1.subset.R1.fastq.paired.fq.gz	${PIPELINE_HOME}/.test/rawdata/iCre_D0_1.subset.R2.fastq.paired.fq.gz
+iCre_D0_2	iCre_D0	${PIPELINE_HOME}/.test/rawdata/iCre_D0_2.subset.R1.fastq.paired.fq.gz	${PIPELINE_HOME}/.test/rawdata/iCre_D0_2.subset.R2.fastq.paired.fq.gz
```

Comment:
does the aspen driver script need to call `envsubst` instead of `sed` on this file since these are bash variables now?

Reply:
looks like this is already taken care of https://github.com/CCBR/ASPEN/blob/8c2c0ab0f09537b9552c66dcf201de84c083f7b5/aspen#L216